src/components/KTooltip/KTooltip.vue

```diff
@@ -11,7 +11,9 @@
     trigger="hover"
     width="auto"
   >
-    <slot />
+    <div tabindex="0">
```

Comment:
question: why the tabindex here?

Reply:
See the Slack thread

Reply:
This actually causes an issue whereby if a tooltip wraps a button, you now have to tab twice to interact with the button. Max is going to add specific logic for the label tooltips to avoid the scenario